modules/apps/asset/asset-tags-admin-web/src/main/resources/META-INF/resources/merge_tag.jsp

```diff
@@ -131,9 +131,17 @@ renderResponse.setTitle(LanguageUtil.get(request, "merge-tags"));
 			);
 
 			if (mergeTagNames.length < 2) {
-				alert(
-					'<liferay-ui:message arguments="2" key="please-choose-at-least-x-tags" />'
-				);
+				if (Liferay.__FF__.customDialogsEnabled) {
+					Liferay.Util.openAlertModal({
+						message:
+							'<liferay-ui:message arguments="2" key="please-choose-at-least-x-tags" />',
+					});
+				}
+				else {
+					alert(
+						'<liferay-ui:message arguments="2" key="please-choose-at-least-x-tags" />'
+					);
```

Comment:
If you wish, you can remove duplication by setting something like
```jsp
const message = '<liferay-ui:message arguments="2" key="please-choose-at-least-x-tags" />';
```

Reply:
I decided not go with just on this PR because this code will be removed sooner once the FF is removed.